tests/test_automation/github-actions/ci/run_step.sh

```diff
@@ -33,11 +33,23 @@ case "$1" in
 
     if [[ "${GH_JOBNAME}" =~ (-CUDA) ]]
     then
-      echo "Set CUDACXX CMake environment variable to nvcc standard location"
-      export CUDACXX=/usr/local/cuda/bin/nvcc
-        
-      # Make current environment variables available to subsequent steps
-      echo "CUDACXX=/usr/local/cuda/bin/nvcc" >> $GITHUB_ENV
+      
+      if [[ "${GH_JOBNAME}" =~ (-AFQMC-Offload-) ]]
```

Comment:
Not critical to change, but I think we only need Offload here. A no AFQMC build would also need this fix.

Reply:
Good point, we have a `Clang12-NoMPI-Offload-Real` job in GitHub Actions hosted runners. The AFQMC-Offload makes it less invasive with the previos, but it's good to keep this things in mind as CI evolves.

Reply:
I think `Clang12-NoMPI-Offload-Real` was for the host. We might better to rename it as `Clang12-NoMPI-Offload-Host-Real`

Reply:
Please also remove AFQMC. Thanks to the outer if on `-CUDA`, offload to host won't be affected.